roles/sahara/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+sahara:
+  enabled: True
```

Comment:
This should default to off. We would rather implementers explicitly turn on "optional" features rather than get them by accident.